sdcm/cluster.py

```diff
@@ -2134,11 +2300,34 @@ def wait_for_init(self):
         pass
 
     def install_scylla_monitoring_prereqs(self, node):
-        prereqs_script = dedent("""
-            yum install -y python-pip unzip wget docker
-            pip install --upgrade pip
-            pip install pyyaml
-        """.format(**locals()))
+        if node.is_rhel_like():
+            prereqs_script = dedent("""
+                yum install -y python-pip unzip wget docker
+                pip install --upgrade pip
+                pip install pyyaml
+            """.format(**locals()))
```

Comment:
Why do you use `.format` if there is nothing to format in the string?
Please remove `.format`